client/src/components/Export/ExportButtons/ExportButtons.js

```diff
@@ -17,7 +17,7 @@ export default class ExportButtons extends React.Component {
             <span>
                 <span>
                     <ExportToCsvButton className="mt-2" fields={csvExportFields}
-                                       data={buildBulkExportData(monuments, csvExportFields, false)}
+                                       data={buildBulkExportData(monuments, csvExportFields, true)}
```

Comment:
This makes export add spaces after commas to be consistent with bulk upload change